lumen/ai/prompts/join_required.jinja2

```diff
@@ -1,6 +1,6 @@
 Determine whether a table join is required to answer the user's query.
 
-The current table `{{ table }}` follows this JSON schema:
+The current table '{{ table }}' follows this JSON schema:
```

Comment:
The LLM is using ` in its query, but it's invalid syntax.

Reply:
Okay please fix on the other PR.

Reply:
See https://github.com/holoviz/lumen/pull/648